assignments/objects.js

```diff
@@ -19,26 +19,77 @@ const example = {
 
 // Write your intern objects here:
 
+const mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const kennan = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  speak: function(){
+    return "Hello, My name is Kennan!";
+  }
+}
+
+const keven = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+
+const gannie = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+
+const antonietta = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F",
+  multiplyNums: function(num1,num2) {
+    return num1 * num2;
+  }
+}
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(mitzi.name);
+
 // Kennan's ID
 
+console.log(keven.id);
+
 // Keven's email
 
+console.log(keven.email);
+
 // Gannie's name
 
+console.log(gannie.name);
+
 // Antonietta's Gender
 
+console.log(antonietta.gender);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
```

Comment:
you could have also done something like a kennan.speak function here and then console log instead of putting it in the object directly.

Reply:
i semi wanted to do it as you were challenging them to do so in the chat last night. 

like 
kennan.speak = function(){ return "I can talk";};

Reply:
No worries, my comment was just a suggestion, not that you did anything wrong. You are doing great! I like that you are rising up to the challenges.